crates/migration/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+pub mod manager;
+
+pub use manager::Manager;
```

Comment:
What's the point of a submodule here if it's the only thing in the crate?

Reply:
Just for consistency with `replication` (`replication::coordinator` / `replication::replica`).
I don't have a strong preference on this, can move if it bothers you.

I was also considering moving those in separate crates: `replication_coordinator`, `replica` and `migration_manager`.

Reply:
It's not bothering me, was just curious.